src/Compilers/VisualBasic/Portable/Syntax/SyntaxKindFacts.vb

```diff
@@ -793,6 +799,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                    {"istrue", SyntaxKind.IsTrueKeyword},
                    {"join", SyntaxKind.JoinKeyword},
                    {"key", SyntaxKind.KeyKeyword},
+                   {"load", SyntaxKind.LoadKeyword},
```

Comment:
At the moment VB compiler expects any keyword that can also be used as an identifier to be added to contextual keywords.
Scanner's `ScanIdentifierOrKeyword` method sets the token kind of all keyword tokens not added to contextual keywords as keywords and not as identifiers (it does not check `IsPreprocessorKeyword`).